src/status_im/utils/slurp.clj

```diff
@@ -17,15 +16,18 @@
       (.mkdir resources))))
 
 (defmacro slurp [file]
-  (if prod?
-    (let [name      (str/replace file #"[\/\.]" "_")
-          file-name (str resources-dir name)]
-      (check-resources-dir)
-      (copy-file file (str file-name "-raw.js"))
-      (let [res (gensym "res")]
-        `(let [~res (atom nil)]
-           (fn []
-             (or @~res
-                 (reset! ~res (js/require ~(str file-name ".js"))))))))
-    `(fn []
-       ~(clojure.core/slurp file))))
+  (clojure.core/slurp file))
+
+#_(defmacro slurp [file]
```

Comment:
could you elaborate?

Reply:
previous macro was for modules, just going back to a simple slurp, that can be called in a module if we want the file to be actually loaded later